platform/android/tts_android.cpp

```diff
@@ -99,6 +149,7 @@ void TTS_Android::_java_utterance_callback(int p_event, int p_id, int p_pos) {
 	if (unlikely(!initialized)) {
 		initialize_tts();
 	}
+	ERR_FAIL_COND(!initialized);
 	ERR_FAIL_NULL(tts);
```

Comment:
For convenience and ensuring things are done consistently it might be worth separating these two lines into a helper macro, like `CHECK_TTS_VALID`, so we don't forget to add either, though it might reduce readability by removing it from scope, alternatively it could be converted into:
```suggestion
	ERR_FAIL_COND_MSG(!initialized || tts == nullptr, "Text to Speech not initialized.");
```
Or some other appropriate error message

Reply:
Macro won't be convenient since some of these check have different return values, changed it to single `COND_[V]_MSG`.